arcaflow_plugin_fio/fio_schema.py

```diff
@@ -408,7 +485,11 @@ def write_jobs_to_file(self, filepath: Path):
             cfg[job.name] = {}
             for key, value in asdict(job.params).items():
                 if value is not None:
-                    cfg[job.name][key] = str(value)
+                    if isinstance(value, (bool, int)):
+                        item_value = int(value)
+                    else:
+                        item_value = str(value)
+                    cfg[job.name][key] = str(item_value)
```

Comment:
Are you sure that you want a mixture of `int` and `str` values?  Perhaps line 489 should be
```python
                        item_value = str(int(value))
```

Reply:
I don't *think* it makes a difference in this case, but it looks like converting the `int` to a `str` in this case also doesn't hurt anything and is more consistent.